eth/catalyst/api.go

```diff
@@ -244,12 +298,11 @@ func decodeTransactions(enc [][]byte) ([]*types.Transaction, error) {
 	return txs, nil
 }
 
-func insertBlockParamsToBlock(params executableData) (*types.Block, error) {
+func InsertBlockParamsToBlock(params ExecutableData) (*types.Block, error) {
```

Comment:
Shouldn't this method enforce `block.Hash() == params.BlockHash` check?

Reply:
Fixed!